hooks/auth.tsx

```diff
@@ -0,0 +1,112 @@
+import * as SecureStore from 'expo-secure-store';
```

Comment:
Note that this isn't being used at the moment. Currently guest auth is hacked in via `useState`.